examples/minimal.rs

```diff
@@ -143,7 +143,7 @@ pub fn run_winit() -> Result<(), Box<dyn std::error::Error>> {
         .unwrap();
 
     std::env::set_var("WAYLAND_DISPLAY", "wayland-5");
-    std::process::Command::new("weston-terminal").spawn().ok();
+    std::process::Command::new(detect_terminal()).spawn().ok();
```

Comment:
The *minimal* example does not need any terminal detection imo. It should be as concise and *minimal* as possible.